src/testIntegration/java/uk/gov/justice/digital/delius/controller/secure/OffendersResource_getOffenderByCrn.java

```diff
@@ -53,6 +54,56 @@ public void canGetOffenderSummaryByCrn() {
     assertThat(offenderDetail.getOtherIds().getCrn()).isEqualTo("X320741");
   }
 
+    @Test
+    public void givenUserIsNotExcluded_thenAccessAllowedWithAllRoles(){
+        final var username = "bernard.beaks";
+        final var path = "/offenders/crn/X440877";
+
```

Comment:
I went to town on the test cases here as it helped me get my head around the problem when I was starting out. For regression purposes it probably isn't needed as the complex logic is also tested at the UserAccessService level as well. Happy to pare these back if it's overkill.